<?xml version="1.0" encoding="UTF-8"?><testrun name="RecursiveDigitSumTest" project="lab12" tests="4" started="4" failures="2" errors="0" ignored="0">
  <testsuite name="lab12.RecursiveDigitSumTest" time="0.0">
    <testcase name="testSumOfDigitsInt" classname="lab12.RecursiveDigitSumTest" time="0.0"/>
    <testcase name="testSumOfDigitsBigInteger" classname="lab12.RecursiveDigitSumTest" time="0.0">
      <failure>java.lang.AssertionError: expected:&lt;126&gt; but was:&lt;135&gt;&#13;
	at junit@1000/org.junit.Assert.fail(Assert.java:89)&#13;
	at junit@1000/org.junit.Assert.failNotEquals(Assert.java:835)&#13;
	at junit@1000/org.junit.Assert.assertEquals(Assert.java:647)&#13;
	at junit@1000/org.junit.Assert.assertEquals(Assert.java:633)&#13;
	at lab12/lab12.RecursiveDigitSumTest.testSumOfDigitsBigInteger(RecursiveDigitSumTest.java:38)&#13;
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at junit@1000/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)&#13;
	at junit@1000/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at junit@1000/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)&#13;
	at junit@1000/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at junit@1000/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)&#13;
	at junit@1000/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)&#13;
	at junit@1000/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)&#13;
	at junit@1000/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)&#13;
	at junit@1000/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)&#13;
	at junit@1000/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)&#13;
	at junit@1000/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)&#13;
	at junit@1000/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at junit@1000/org.junit.runners.ParentRunner.run(ParentRunner.java:413)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:530)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:758)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:453)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:211)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNullInputInteger" classname="lab12.RecursiveDigitSumTest" time="0.0">
      <failure>java.lang.AssertionError: Expected exception: java.lang.IllegalArgumentException&#13;
	at junit@1000/org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:34)&#13;
	at junit@1000/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)&#13;
	at junit@1000/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)&#13;
	at junit@1000/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)&#13;
	at junit@1000/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)&#13;
	at junit@1000/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)&#13;
	at junit@1000/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)&#13;
	at junit@1000/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)&#13;
	at junit@1000/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)&#13;
	at junit@1000/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at junit@1000/org.junit.runners.ParentRunner.run(ParentRunner.java:413)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:530)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:758)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:453)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:211)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testNullInputBigInteger" classname="lab12.RecursiveDigitSumTest" time="0.0"/>
  </testsuite>
</testrun>
